.github/workflows/selfdrive_tests.yaml

```diff
@@ -181,7 +180,7 @@ jobs:
       run: |
         ${{ env.RUN }} "source selfdrive/test/setup_xvfb.sh && \
                         export MAPBOX_TOKEN='pk.eyJ1Ijoiam5ld2IiLCJhIjoiY2xxNW8zZXprMGw1ZzJwbzZneHd2NHljbSJ9.gV7VPRfbXFetD-1OVF0XZg' && \
-                        $PYTEST --timeout 40 -m 'not slow' && \
+                        $PYTEST --timeout 40 -m 'not slow' -n $(nproc) && \
```

Comment:
why was this changed from auto only here?

Reply:
the pytest auto only returns 4 even though there are 8 cpus (perhaps something with hyperthreading or docker), im going to bring it up with the namespace people.

Reply:
`-n auto` returns physical cores, maybe there's just 4 hyperthreaded cores. (using logical cores still improves performance, so maybe we should switch to `-n logical`)